### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by neringamaj**
Something like that?

### CODE REVIEW

File: BlazorApp2/README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
+RIDALOT is a web app which helps users find people, who will transport big objects or massive amounts of goods to the nearest waste site. We are focused on finding those who want to earn extra money and already own the right equipment. You can post your task here and get results immediately.
+
+Product Features: 
+- Find people who will help transport any object.
+- No fees required.
+- Easy to use.
+- Available 24/7.
+- Add multiple tasks to increase earnings.
+- User ratings & reviews.
+- Free registration required.
```
Comment (Astijq) [inline BlazorApp2/README.md:10]: so just free registration?

File: BlazorApp2/README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
-RIDALOT
-write description
-Astijus description
+RIDALOT is a web app which helps users find people, who will transport big objects or massive amounts of goods to the nearest waste site. We are focused on finding those who want to earn extra money and already own the right equipment. You can post your task here and get results immediately.
+
+Product Features: 
+- Find people who will help transport any object.
+- No fees required.
```
Comment (Astijq) [inline BlazorApp2/README.md:5]: yes required

File: BlazorApp2/README.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
-RIDALOT
-write description
-Astijus description
+RIDALOT is a web app which helps users find people, who will transport big objects or massive amounts of goods to the nearest waste site. We are focused on finding those who want to earn extra money and already own the right equipment. You can post your task here and get results immediately.
```
Comment (Astijq) [inline BlazorApp2/README.md:1]: Also we can add hazardous waste

**Review by Astijq — CHANGES_REQUESTED** [review]
needs a little more

**Comment by neringamaj** [comment]
I made changes

**Review by Astijq — APPROVED** [review]

**Pull request merged**